homeTask_4/homeTask_4/Program.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+
+namespace homeTask_4
+{
+    class Program
+    {
+
+        
+//1. Создать два произвольных массива, подсчитать максимальное и минимальное значение их элементов.Вывести на экран.
+//2. Поменять в массивах максимальные и минимальные значения местами
+        static void InitializeArray(int[] array)
+        {
+            Random rnd = new Random();
+            for (int i = 0; i < array.Length; ++i)
+            {
+                array[i] = rnd.Next(1, 50);
+            }
+        }
+
+        static void PrintArray(int[] array)
+        {
+            for (int i = 0; i < array.Length; ++i)
+            {
+                Console.WriteLine("№={0} is {1}", i + 1, array[i]);
+            }
+        }
+
+        static int MaxElement(int[] array)
```

Comment:
i think  we can simplify 2 methods `MaxElement` and `MinElement` in one method like `MinMaxElement`.
you have 2 ways:
1 - return array of 2 element in which 1 element - min, 2 - max
2 - add flag in method's parameters like bool `isMinCalculation`  and return min/max value depends on flag.

Because it's looks like code duplication.

Reply:
Aha, i see your point. I will rework it and will choose the most suitable method for me.